packages/sdk/lib/browser.ts

```diff
@@ -2,6 +2,7 @@
 import {
   Itly as ItlyBase,
   Options,
+  LoadOptions,
```

Comment:
Need to also export `LoadOptions` in `packages/sdk/lib/node.ts`

Reply:
Done